README.md

```diff
@@ -794,6 +843,52 @@ authClient.fingerprint()
 })
 ```
 
+### `loginRedirect(fromUri, additionalParams)`
+
+> :hourglass: async
+
+Performs a full-page redirect to Okta with optional request parameters.
+
+The `additionalParams` are mapped to Okta's [`/authorize` request parameters](https://developer.okta.com/docs/api/resources/oidc#authorize). This will override any existing [configuration](#configuration-options). As an example, if you have an Okta `sessionToken`, you can bypass the full-page redirect by passing in this token.
```

Comment:
They are "mapped" but we have our own version of these parameters which are in camel-case. We may also add additional params that are not known to the backend. Here's the reference for the params: https://github.com/okta/okta-auth-js#token